api/types/types.go

```diff
@@ -169,6 +170,9 @@ func (pi *PacketInjection) Validate() error {
 	if _, ok := allowedTypes[pi.Type]; !ok {
 		return errors.New("given type is not supported")
 	}
+	if pi.TTL < 0 || pi.TTL > 255 {
```

Comment:
this test is not useful as the type is uint8

Reply:
removed